src/NServiceBus.Persistence.ServiceFabric.AcceptanceTests/AcceptanceTests.csproj

```diff
@@ -1,145 +1,48 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <PlatformTarget>x64</PlatformTarget>
+    <OutputType>Exe</OutputType>
```

Comment:
What's the reason to make `<OutputType>` an exe rather than dll?

Reply:
Because before the project system change, these were exes, and all of the sfproj xml metadata references an exe assembly. I think this was missed from the original conversion.

Reply:
@bording something is off. While the project was of type `Microsoft.NET.Sdk.Web`, did it actually create exes? I'm pretty sure it was creating DLLs for the test projects.

Reply:
It needs to be an exe, remember this is executable that hosts the service https://github.com/Particular/NServiceBus.Persistence.ServiceFabric/blob/develop/src/NServiceBus.Persistence.ServiceFabric.AcceptanceTests/Program.cs#L18